py3status/modules/sysdata.py

```diff
@@ -326,6 +348,14 @@ def sysData(self):
             self.values['swap_unit'] = swap_unit
             self.py3.threshold_get_color(swap_used_percent, 'swap')
 
+        # get load average
+        if self.py3.format_contains(self.format, 'load*'):
+            load1, load5, load15 = self.data.load()
+            self.values['load1'] = load1
+            self.values['load5'] = load5
+            self.values['load15'] = load15
+            self.py3.threshold_get_color(load1, 'load')
```

Comment:
The problem is that the explanation above works if you have one core. If you have two cores, a load of 2 is perfectly fine, it means each of your core had just enough work to keep them busy. You get the idea. Load is the average number of active process at once over the specified amount of time. So it's really a number of processes. If we really want to see it as percentage, we have to do `load * 100 / n_cpus`. And I really don't think it'll match people expectations. If for everything else, the threshold is compared against the value and for this value only it's compared against the load percent per cpu, that's going to surprise people.